lib/gis/open.c

```diff
@@ -21,6 +21,7 @@
 
 #include <grass/gis.h>
 #include <grass/glocale.h>
+#include <grass/vect/dig_defines.h>
```

Comment:
This is not allowed. Vector headers can only be included by vector libs or libs depending on vector libs or modules.